### ISSUE TITLE
Google Analytics causing summarize to not work properly

### ISSUE THREAD

**Issue by salsakran** [metabase/metabase#11137]
**Describe the bug**
When trying to change the date granularity or remove a metric (eg, from the default starting point of users + page views over time), clicking on the granularity or the "x" doesn't do anything. 

**Logs**
JS logs
```
Uncaught (in promise) TypeError: Cannot read property 'query' of null
    at t.value (app-main.bundle.js?4718c50f716c2728dc8c:5)
    at t.value (app-main.bundle.js?4718c50f716c2728dc8c:5)
    at Array.value (app-main.bundle.js?4718c50f716c2728dc8c:5)
    at t.value (app-main.bundle.js?4718c50f716c2728dc8c:5)
    at app-main.bundle.js?4718c50f716c2728dc8c:5
    at Array.map (<anonymous>)
    at t.value (app-main.bundle.js?4718c50f716c2728dc8c:5)
    at app-main.bundle.js?4718c50f716c2728dc8c:5
    at r (app-main.bundle.js?4718c50f716c2728dc8c:5)
    at t.n.value (app-main.bundle.js?4718c50f716c2728dc8c:5)
```
**To Reproduce**
Go to browse data, click on a GA property. 
You'll see a prefilled query. 
Try to change the date granularity , and nothing happens.

**Expected behavior**
I'd expect the date granularity to change

**Information about your Metabase Installation:**

- Your browser and the version: Chrome
- Your operating system: MacOS
- Your databases: GA
- Metabase version: Current msater branch 207e8d6
- Metabase hosting environment: Beanstalk
- Metabase internal database: PostgreSQL/RDS

**Severity**
This makes GA unusable with Metabase and I reverted to using GA directly.

**Comment by mbiberhofer** [comment]
Greetings,

We started using metabase only recently, using 0.33.4 as a baseline. We experienced the following error when trying to create a simple question on our GA database:

**Chrome:**

```
app-main.bundle.js?3f8f0db4d958de85c2e9:5 Uncaught (in promise) TypeError: Cannot read property 'query' of null
    at t.value (app-main.bundle.js?3f8f0db4d958de85c2e9:5)
    at t.value (app-main.bundle.js?3f8f0db4d958de85c2e9:5)
    at Array.value (app-main.bundle.js?3f8f0db4d958de85c2e9:5)
    at t.value (app-main.bundle.js?3f8f0db4d958de85c2e9:5)
    at app-main.bundle.js?3f8f0db4d958de85c2e9:5
    at Array.map (<anonymous>)
    at t.value (app-main.bundle.js?3f8f0db4d958de85c2e9:5)
    at app-main.bundle.js?3f8f0db4d958de85c2e9:5
    at r (app-main.bundle.js?3f8f0db4d958de85c2e9:5)
    at t.n.value (app-main.bundle.js?3f8f0db4d958de85c2e9:5)
```


**Firefox:**

`TypeError: this.table is null        app-main.bundle.js:5:700470`


We saw this ticket and thought it might reflect our issue, so we  decided to simply wait until the fix was released. Today we noticed that version 0.33.5 was released, updated our metabase instance, cleared all caches and cookies and are still getting the same error in both browsers respectively when constructing a simple or custom question on our GA database:

Note that, when we applied the reproduction steps mentioned in the original post on 0.33.4, we could reproduce the above error message. On metabase 0.33.5, my firefox reports:

`TypeError: e is undefined`

When clicking on the "+ Summarize" Button, I'm getting: (original domain replaced with <DOMAIN>)

```
TypeError: e is undefined app-main.bundle.js:5:173252
    value https://metabase.<DOMAIN>/app/dist/app-main.bundle.js?2beff2a9a8b48c4fbba5:5
    value https://metabase.<DOMAIN>/app/dist/app-main.bundle.js?2beff2a9a8b48c4fbba5:5
    r https://metabase.<DOMAIN>/app/dist/app-main.bundle.js?2beff2a9a8b48c4fbba5:5
    value https://metabase.<DOMAIN>/app/dist/app-main.bundle.js?2beff2a9a8b48c4fbba5:5
    getQuery https://metabase.<DOMAIN>/app/dist/app-main.bundle.js?2beff2a9a8b48c4fbba5:5
    c https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    i https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    l https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    i https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    P https://metabase.<DOMAIN>/app/dist/app-main.bundle.js?2beff2a9a8b48c4fbba5:5
    i https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    p https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    i https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    run https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    componentWillReceiveProps https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    updateComponent https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    receiveComponent https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    receiveComponent https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    _updateRenderedComponent https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    _performComponentUpdate https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    updateComponent https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    receiveComponent https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    receiveComponent https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    _updateRenderedComponent https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    _performComponentUpdate https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    updateComponent https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    receiveComponent https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    receiveComponent https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    _updateRenderedComponent https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    _performComponentUpdate https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    updateComponent https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    receiveComponent https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    receiveComponent https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    _updateRenderedComponent https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    _performComponentUpdate https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    updateComponent https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    receiveComponent https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    receiveComponent https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    _updateRenderedComponent https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    _performComponentUpdate https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    updateComponent https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    performUpdateIfNecessary https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    performUpdateIfNecessary https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    b https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    perform https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    perform https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    perform https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    w https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    w self-hosted:1025
    closeAll https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    perform https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    batchedUpdates https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    batchedUpdates https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    dispatchEvent https://metabase.<DOMAIN>/app/dist/vendor.bundle.js?2beff2a9a8b48c4fbba5:1
    dispatchEvent self-hosted:1069
```

So basically, nothing related to GA currently works. :-) (Note that in our case that's not an urgent issue; we're currently evaluating metabase for future use)

Anyone else who can reproduce this issue? Should I open a new ticket, or does anyone experienced in developing metabase confirm/agree that this is the same issue?

Cheers & thanks for your hard work,
Mario

**Comment by flamber** [comment]
@mbiberhofer Are you using Docker? If so, then expect a re-release or maybe a new version #11289

**Comment by mbiberhofer** [comment]
@flamber I am indeed running metabase on a docker swarm cluster, using: https://hub.docker.com/layers/metabase/metabase/v0.33.5/images/sha256-bf5604fd7b545eb498ff466fc0609e596159c33b3ba5a136499ba5aebcc4bbd6

... so yeah, I guess I'm affected by #11289 -- I'll wait patiently until the tag's been refreshed/the image's been rebuilt correctly. Thanks for the Info!

**Comment by mbiberhofer** [comment]
I just applied the hotfix for the container according to https://github.com/metabase/metabase/issues/11289#issuecomment-552909619 (verified SHA512 sum of running binary: `7f318f1bbbab4723500aac6c719957cad14d342efa5656250114cd93a1e4dc39  /app/metabase.jar`)

Sadly, the issue seems to prevail: When creating a simple question and selecting my GA database as data source, nothing happens and `TypeError: this.table is null` is displayed in the console (Firefox 70.0.1). The same happens when creating a custom query, adding/changing filters.

**Comment by mbiberhofer** [comment]
Good new everyone!

The newly released docker-image (hotfix-version 0.33.5.1 / see https://github.com/metabase/metabase/issues/11289#issuecomment-553630053 ) seems to fix this sissue -- at least, I can't reproduce it anymore. :-)

Thanks for your hard work, I think we can keep this issue closed.
